test/AddCommandTest.py

```diff
@@ -200,7 +200,7 @@ def test_add_dep9(self):
         command = ListCommand.ListCommand(["Bar"], self.todolist, self.out, self.error)
         command.execute()
 
-        self.assertEqual(self.output, "|kbn| {today} Bar p:1 +Project\n|kbn| {today} Bar +Project\n".format(today=self.today))
+        self.assertEqual(self.output, "|kbn| {today} Bar p:1 +Project\n|kbn|   {today} Bar +Project\n".format(today=self.today))
```

Comment:
Why these additional spaces? They are added in [this commit](https://github.com/MinchinWeb/topydo/commit/b1b1d4f8969c8d1f93399b9e5a8e52374646f28a) under "random style improvements". Same remark for the diff below.